maven-model-builder/src/main/java/org/apache/maven/model/validation/DefaultModelValidator.java

```diff
@@ -1489,8 +1491,10 @@ private boolean validateModelVersion(
                     Version.V20,
                     "modelVersion",
                     null,
-                    "of '" + string + "' is newer than the versions supported by this version of Maven: " + values
-                            + ". Building this project requires a newer version of Maven.",
+                    "of '" + string + "' is newer than the versions supported by this Maven version("
+                            + getMavenVersion()
+                            + ").\n Supported modelVersions are: " + values
+                            + ".\n Building this project requires a newer version of Maven.",
```

Comment:
I do not know the details of how the message is used, but I presume that `\n` should be `System.lineSeparator()`. Maybe there is no space at the beginning of the new line?